sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -66,6 +66,20 @@ For the event processor client to make use of Azure Storage blobs for checkpoint
 
 For more concepts and deeper discussion, see: [Event Hubs Features](https://docs.microsoft.com/azure/event-hubs/event-hubs-features).
 
+### Thread safety
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
+
+### Additional concepts
+<!-- CLIENT COMMON BAR -->
```

Comment:
The examples and concepts in these sections are not relevant to the messaging libraries due to the references to the `Azure.Core` pipeline and HTTP specifics.  

Please remove Event Hubs and Service Bus from this PR and open issues to address.   I'll go through and map the relevant parts of this to the samples to reflect the correct context and semantics.   _(if you would, please either include this section or a link to a README with it so that I can reference as an example)_

//cc: @AlexGhiondea

Reply:
I think this is a fair comment. We should probably remove the pieces that are not relevant to a specific library.

Reply:
Reverted Event Hubs and Service Bus changes. Here's the link to the issue: https://github.com/Azure/azure-sdk-for-net/issues/18548.

Reply:
Thanks, @kinelski!